[17.0][MIG] l10n_es_mis_report: Migration to 17.0

mpascuall opened the pull request:
Module migrated to version 17.0

cc https://github.com/APSL 154488

@miquelalzanillas @lbarry-apsl @javierobcn @peluko00 please review

Depends on:

- [x] <https://github.com/OCA/mis-builder/pull/607>

JordiBForgeFlow committed:
[ADD] l10n_es_mis_reports: MIS reports for PL / BS

JordiBForgeFlow committed:
l10n_es_mis_report - compatibilizar con modalidad de no-expansión
de cuentas de mis_builder.

JordiBForgeFlow committed:
Corrige algunos conceptos del balance llevando
ciertas cuentas al pasivo o activo según el signo
del saldo en lugar de llevar todo el debe y todo el haber.

pedrobaeza committed:
[IMP] l10n_es_mis_report: Use pbale/nbale

For the balance sections where the sum of debtor/creditor balances is needed,
with this new expression we can get the interpreted result.

JordiBForgeFlow committed:
l10n_es_mis_report. Nunca autoexpandir la sección VII. Resultado del ejercicio

JordiBForgeFlow committed:
remove es40000

oca-travis committed:
[UPD] Update l10n_es_mis_report.pot

pedrobaeza committed:
[IMP] l10n_es_mis_report: Add 473 to abbreviated balance

According discussion in #970, adding the account to the template.

pedrobaeza committed:
[MIG] l10n_es_mis_report: Migration to 12.0

* Standard procedure
* README by fragments

antoniocanovas committed:
[IMP] l10n_es_mis_report: Añadidos informes MIS para entidades SFL (#1074)

MiquelRForgeFlow committed:
[IMP] l10n_es_mis_report: black, isort

MiquelRForgeFlow committed:
[MIG] l10n_es_mis_report: Migration to 13.0

danimv5 committed:
Added translation using Weblate (Spanish)

pedrobaeza committed:
[IMP] l10n_es_mis_report: Unfold 746 in 2 accounts

Following odoo/odoo#55522, here we transfer that unfolding to the MIS reports.

Closes #1434

HaraldPanten committed:
[MIG]l10n_es_mis_report: Migration to 14.0

Reyes4711-S73 committed:
[14.0][FIX] l10n_es_mis_report: expresión incorrecta

pedrobaeza committed:
[FIX] l10n_es_mis_report: Fix 1146 account

The account code 1146 [1] is not yet on the Spanish CoA, but it was created
due to a 2014 law [2] for a special provision case.

By chance, a mistake putting the same balance code as expression, that
inexisting account was mapped to an incorrect place.

On this commit, we remove the incorrect expression, and add a new section
under "III. Reservas" for reflecting this balance according [3].

[1] https://www.supercontable.com/informacion/Contabilidad/Cuenta_1146_Reserva_de_Capitalizacion..html
[2] https://www.supercontable.com/informacion/impuesto_sociedades/Articulo_25_Ley_27-2014-_de_27_de_noviembre-_del_.html
[3] https://www.supercontable.com/envios/articulos/BOLETIN_SUPERCONTABLE_12_2017_Contenido_General_2.htm

Commit:
[WIP][14.0] Sync module module l10n_es_mis_report

Commit:
[IMP] l10n_es_mis_report: black, isort, prettier

Commit:
[MIG] l10n_es_mis_report: Migration to 15.0

juliap98 committed:
Translated using Weblate (Catalan)

Currently translated at 12.1% (37 of 304 strings)

Translation: l10n-spain-15.0/l10n-spain-15.0-l10n_es_mis_report
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-15-0/l10n-spain-15-0-l10n_es_mis_report/ca/

manuelregidor committed:
[15.0][IMP]l10n_es_mis_report: actualizado 'balance de pymes' y 'perdidas y ganancias de pymes'con nuevo formato de registro mercantil

jabelchi committed:
Translated using Weblate (Catalan)

Currently translated at 100.0% (310 of 310 strings)

Translation: l10n-spain-15.0/l10n-spain-15.0-l10n_es_mis_report
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-15-0/l10n-spain-15-0-l10n_es_mis_report/ca/

LudLaf committed:
Translated using Weblate (Spanish)

Currently translated at 10.0% (31 of 310 strings)

Translation: l10n-spain-15.0/l10n-spain-15.0-l10n_es_mis_report
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-15-0/l10n-spain-15-0-l10n_es_mis_report/es/

pedrobaeza committed:
[MIG] l10n_es_mis_report: Migration to 16.0

Commit:
[UPD] Update l10n_es_mis_report.pot

OCA-git-bot committed:
[UPD] README.rst

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 10.3% (32 of 310 strings)

Translation: l10n-spain-16.0/l10n-spain-16.0-l10n_es_mis_report
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-16-0/l10n-spain-16-0-l10n_es_mis_report/es/

OCA-git-bot committed:
[UPD] README.rst

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (310 of 310 strings)

Translation: l10n-spain-16.0/l10n-spain-16.0-l10n_es_mis_report
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-16-0/l10n-spain-16-0-l10n_es_mis_report/es/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-spain-16.0/l10n-spain-16.0-l10n_es_mis_report
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-16-0/l10n-spain-16-0-l10n_es_mis_report/

Commit:
[FIX] l10n_es_mis_report: Corrección cuentas 7460% y 7461% en PyG abreviado.

OCA-git-bot committed:
[BOT] post-merge updates

peluko00 reviewed (approved):
Works good, tested in runboat

javierobcn reviewed (approved):
Works well

angelmoya commented on test-requirements.txt:
```diff
@@ -1 +1,2 @@
 odoo_test_helper
+odoo-addon-mis_builder @ git+https://github.com/OCA/mis-builder.git@refs/pull/607/head#subdirectory=mis_builder
```
esto es correcto?

mpascuall replied:
Sí, depende de esta pr.

angelmoya replied:
lo digo por el error en tests:
Run for reqfile in requirements.txt test-requirements.txt ; do
odoo-addon-mis_builder @ git+https://github.com/OCA/mis-builder.git@refs/pull/607/head#subdirectory=mis_builder
Unreleased dependencies found in test-requirements.txt.
Error: Process completed with exit code 1.

pedrobaeza replied:
Ángel, echa un vistazo a esta referencia para entenderlo: 

https://github.com/OCA/maintainer-tools/wiki/Use-temporary-reference%28s%29-to-another-pull-request%28s%29

ValentinVinagre commented:
@manuelregidor @Tisho99 could u review please?

HaraldPanten commented:
/ocabot migration l10n_es_mis_report

manuelregidor commented:
Wouldn't it be better to put the changes related to the readme files in the module l10n_es_vat_book in an idependent commit?

manuelregidor reviewed (approved):
Technical review. LGTM

mpascuall commented:
> Wouldn't it be better to put the changes related to the readme files in the module l10n_es_vat_book in an idependent commit?

Fixed that. Changes in l10n_es_vat_book are now deleted

HaraldPanten commented:
@mpascuall Puedes hacer rebase?

pedrobaeza commented:
El rebase no es correcto y hay conflictos.

mpascuall committed:
[IMP] l10n_es_mis_report: pre-commit auto fixes

mpascuall committed:
[MIG] l10n_es_mis_report: Migration to 17.0

pedrobaeza commented on l10n_es_mis_report/__manifest__.py:
```diff
@@ -8,7 +8,7 @@
     "website": "https://github.com/OCA/l10n-spain",
     "category": "Reporting",
-    "version": "16.0.1.1.0",
+    "version": "17.0.1.1.0",
```
Tiene que ser 17.0.1.0.0

pedrobaeza reviewed (approved):
/ocabot merge nobump
